spec/requests/v1/ont/pools_spec.rb

```diff
@@ -55,11 +55,11 @@
     end
 
     context 'pagination' do
-      let!(:expected_pools) { create_list(:ont_pool, 2) }
+      let!(:expected_pools) { create_list(:ont_pool, 2, created_at: Time.zone.now + 10) }
 
       before do
         # There should be 4 pools total so we should get the 2 we just created
-        get "#{v1_ont_pools_path}?page[number]=2&page[size]=2",
+        get "#{v1_ont_pools_path}?page[number]=1&page[size]=2",
```

Comment:
We can tell they are the correct order since the latest pools were returned first which means the created_at descending order is correct. It would be nice to make a shared test for paginators but each pagniated resource will have different response attributes, filters and potentially sort order.